packages/account/src/Components/poi/idv-document-submit/__tests__/idv-document-submit.spec.tsx

```diff
@@ -129,15 +136,15 @@ describe('<IdvDocumentSubmit/>', () => {
         fireEvent.change(document_type_input, { target: { value: 'Test document 2 name' } });
         expect(document_number_input).toBeEnabled();
         expect(screen.queryByText(/please enter the correct format/i)).not.toBeInTheDocument();
-
+        (isDocumentNumberValid as jest.Mock).mockReturnValueOnce('please enter your document number');
         fireEvent.blur(document_number_input);
         expect(await screen.findByText(/please enter your document number/i)).toBeInTheDocument();
 
-        fireEvent.keyUp(document_number_input);
+        (isDocumentNumberValid as jest.Mock).mockReturnValueOnce('please enter the correct format');
         fireEvent.change(document_number_input, { target: { value: 'A-32523' } });
         expect(await screen.findByText(/please enter the correct format/i)).toBeInTheDocument();
 
-        fireEvent.change(document_number_input, { target: { value: '5436454364243' } });
+        fireEvent.change(document_number_input, { target: { value: '5436454364234' } });
```

Comment:
lets use userevent?

Reply:
Unfortunately , I couldn’t change all events to userEvent, I had some issues with mocking returning values do isDocumentNumberValid.
so I left fireEvent in two places

Reply:
isnt this working with 
 userEvent.type(document_number_input, '5436454364243');